.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+<!-- Add the component name to the PR's title. Example: pipelines: Fixed broken link in Getting Started with Kubeflow Pipelines -->
```

Comment:
Can we create GitHub action that will check the PR title to make sure it aligns with the rules ?
I think, KFP has same check for their PR names:
https://github.com/kubeflow/pipelines/tree/master/.github/workflows

Reply:
created issue https://github.com/kubeflow/website/issues/4002